react/index.js

```diff
@@ -1,18 +1,18 @@
-import React, { Component, Fragment } from 'react'
+import './global.css'
```

Comment:
Relative imports should come after the absolute ones. :pick:

Reply:
You sure? Typescript Hero organizes it that way. I didn't see any pattern about the imports order besides the natural language one.

Reply:
yep, it's what we are adopting

Reply:
It organizes the "full" imports first. Like this CSS import that has no attribution. The relative imports (the ones with "attribution") come right after. Then, we have the absolute imports.

Reply:
the way the other apps are structured, absolute imports come first, then the relative "with attribution", then the ones without it, like this CSS one.

Reply:
Another cool extension to disable. Thank you :)

Reply:
Good to know that has no use of alphabetic order in the other apps too. Gr8 m8